lua/core/mappings.lua

```diff
@@ -43,16 +43,6 @@ M.misc = function()
       map("n", "<leader>j", ":m .+1<CR>==", opt)
       map("v", "J", ":m '>+1<CR>gv=gv", opt)
       map("v", "K", ":m '<-2<CR>gv=gv", opt)
-      map("n", "<leader>lf", ":Lspsaga lsp_finder<CR>", opt)
```

Comment:
Hmmm, maybe I've missed it, but why is this PR removing these mappings?

Think `lsp-saga` has been removed from `NvChad:main`, so consider dropping this commit?

(use the same link to our docs as , with `drop`)